lambda-notes/src/components/notes-container/EditBar.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+
+
+const EditBar = props => {
+    return (
+    <div>
+    <Link to={`/EditNoteForm/${props.id}`}>edit</Link>
+    <Link to='/' onClick={event => props.deleteNote(event,props.id)}>delete</Link>
```

Comment:
one approach to these so that they behave like proper buttons would be to use <button> tags, and then use `this.props.history.push('/')` inside the click handler.

If you got an error that history is not defined, you could use:
import { withRouter } from 'react-router';
and at the end: export withRouter(EditBar);